telemetry/csharp/AwsToolkit.Telemetry.Events.Tests/Generated/SupplementalCode.cs

```diff
@@ -7,8 +7,8 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
```

Comment:
You don't need to review the changes to this file. It is produced by the generator and used in some tests.

Reply:
it's useful to see the diff so I'm glad it's in source control